filebeat/tests/system/config/filebeat.yml.j2

```diff
@@ -32,10 +32,10 @@ filebeat.prospectors:
   {%- if prospector_processors %}
   processors:
     {%- for processor in prospector_processors %}
-    {%- for name, settings in processor.iteritems() %}
+    {%- for name, settings in processor.items() %}
```

Comment:
Both Python 2 and 3 have `.items()`. If adding an `import` within `{% %}` is acceptable, `six.iteritems` could be used.